### PR TITLE
javascript-bank | Kianna Brandt

### PR DESCRIPTION
**Pull request by KiannaBrandt**
Screen Shot 2021-03-11 at 3 46 00 PM

### CODE REVIEW

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@KiannaBrandt Please include a screenshot(s) of your successful test output in the pull request.

**Comment by KiannaBrandt** [comment]
Screen Shot 2021-03-11 at 3 46 00 PM

File: javascript-bank/bank.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    this.accounts.push(account);
+    this.nextAccountNumber ++;
+    return account.number;
+  } else {
+    return null;
+  };
+};
+
+Bank.prototype.getAccount = function(number) {
+  for (i = 0; i < this.accounts.length; i++) {
```
Comment (TimelikeClosure) [inline javascript-bank/bank.js]: Please declare all your variables before using them.

File: javascript-bank/bank.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    };
+  };
+  return null;
+};
+
+Bank.prototype.getTotalAssets = function() {
+  var total = 0;
+  for (i = 0; i < this.accounts.length; i++) {
```
Comment (TimelikeClosure) [inline javascript-bank/bank.js]: Same as above.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**